pkg/monitor/azure/nsg/nsg.go

```diff
@@ -109,8 +109,15 @@ func (n *NSGMonitor) toSubnetConfig(ctx context.Context, subnetID string) (subne
 	return subnetNSGConfig{prefixes, subnet.Properties.NetworkSecurityGroup}, nil
 }
 
+// These to be used as a member of a set implemented with a map
+type void struct{}
+
+var setMember = void{}
```

Comment:
no need for this.  just make a `map[string]bool` or `map[string]struct{}`.

Reply:
done.  having a type was clearer but apparently it also cluttered.